From 33dc77fda8c5d993abee4e6c31c7526408877003 Mon Sep 17 00:00:00 2001
From: Martin Geisler <martin@geisler.net>
Date: Wed, 11 Sep 2019 11:17:36 +0100
Subject: [PATCH] markdown_deps: handle test output from toml 0.5.3

The latest minor version of the toml crate seems to give more detailed
error output.
---
 src/markdown_deps.rs | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/src/markdown_deps.rs b/src/markdown_deps.rs
index 00ca12b..a557c1c 100644
--- a/src/markdown_deps.rs
+++ b/src/markdown_deps.rs
@@ -276,10 +276,11 @@ mod tests {
         let block = "[dependencies]\n\
                      foobar = 1.5.8";
         let request = extract_version_request("foobar", block);
-        assert_eq!(
-            request.unwrap_err(),
-            "TOML parse error: expected newline, found a period at line 2"
-        );
+        // toml 0.5.3 returns "found a period at line 2 column 13.
+        // Update the test when we bump the toml crate dependency.
+        assert!(request
+            .unwrap_err()
+            .contains("TOML parse error: expected newline, found a period at line 2"));
     }
 
     #[test]
-- 
2.24.0

